packages/client/e2e/mpe/test-e.spec.ts

```diff
@@ -0,0 +1,245 @@
+}
+
+async function exportMpeRoomToMtvRoom({
+    page,
+    mtvRoomName,
+}: {
+    page: Page;
+    mtvRoomName: string;
+}): Promise<void> {
+    await openMpeSettingsModal({
+        page,
+    });
+
+    const exportToMtvButton = page.locator('text="Export to MTV"');
+    await exportToMtvButton.click();
+
+    await expect(
+        page.locator('text="What is the name of the room?"'),
+    ).toBeVisible();
+
+    await page.fill('css=[placeholder="Francis Cabrel OnlyFans"]', mtvRoomName);
+
+    // Go to opening status screen.
+    await hitGoNextButton({ page });
+
+    // Go to physical constraints screen.
+    await hitGoNextButton({ page });
+
+    // Go to playing mode screen.
+    await hitGoNextButton({ page });
+
+    // Go to vote constraints screen.
+    await hitGoNextButton({ page });
+
+    // Go to confirmation screen.
+    await hitGoNextButton({ page });
+
+    // Confirm export.
+    await hitGoNextButton({ page });
+
+    const roomNameInFullScreenPlayer = page.locator(
+        withinMusicPlayerFullScreen(`text="${mtvRoomName}"`),
+    );
+    await expect(roomNameInFullScreenPlayer).toBeVisible();
+}
+
+// Test-E USerA MTV and MPE Transversal
+// UserA creates an MTV room, default settings.
+// UserA hit play control button in the MTV room
+// UserA creates an MPE room, default settings.
+// UserA adds a song to the created MPE room
+// UserA exports the created MPE as MTV, default settings, created MTV should still be playing, all mpe tracks should have been exported
+// UserA should see exported MTV room music player fullscreen
+test('mpe e2e test-e', async ({ browser }) => {
+    const { page } = await setupAndGetUserPage({
+        browser,
+        knownSearches,
+        userIndex: 0,
+    });
+
+    await createRoom({
+        creatorPage: page,
+    });
+
+    await hitMtvPlayButton({
+        page,
+    });
+
+    await assertMusicPlayerStatusIs({
+        page,
+        testID: 'music-player-playing-device-emitting',
+    });
+
+    const minimizeMusicPlayerButton = page.locator(
+        `css=[aria-label="Minimize the music player"]`,
+    );
+    await expect(minimizeMusicPlayerButton).toBeVisible();
+    await minimizeMusicPlayerButton.click();
+
+    await goToHomeTabScreen({
+        page,
+    });
+
+    await createMpeRoom({
+        page,
+    });
+
+    await assertMusicPlayerStatusIs({
+        page,
+        testID: 'music-player-playing-device-emitting',
+    });
+
+    const addedTrack = await addTrack({
+        page,
+        searchQuery: 'BB Brunes',
+    });
+
+    await assertMusicPlayerStatusIs({
+        page,
+        testID: 'music-player-playing-device-emitting',
+    });
+
+    await exportMpeRoomToMtvRoom({
+        mtvRoomName: 'cocorico',
+        page,
+    });
+
+    const addedTrackTestID = `mtv-${addedTrack.id}-track-card-container`;
+    const addedTrackMtvCard = page.locator(
+        withinMpeRoomScreen(`css=[data-testid$="${addedTrackTestID}"]`),
```

Comment:
I think we want to assert that the song can be found on the Music Player? Not in the MPE room?